### PR TITLE
Update rc client

### PR DESCRIPTION
**Pull request by zhx828**

### CODE REVIEW

File: src/shared/api/generated/CBioPortalAPI-docs.json [hunk 1]
```diff
@@ -2324,6 +2328,43 @@
         }
       }
     },
+    "/structuralvariant/fetch": {
```
Comment (zhx828) [inline src/shared/api/generated/CBioPortalAPI-docs.json]: @inodb  this seems new and not related to my code. You know anything about this?

File: src/shared/api/generated/CBioPortalAPI-docs.json [hunk 2]
```diff
@@ -4488,10 +4488,6 @@
         "hugoGeneSymbol": {
           "type": "string"
         },
-        "length": {
```
Comment (zhx828) [inline src/shared/api/generated/CBioPortalAPI-docs.json]: This length?

File: src/shared/api/generated/CBioPortalAPI-docs.json (hunk 1 above)
Comment (inodb) [reply src/shared/api/generated/CBioPortalAPI-docs.json]: it's this one i think https://github.com/cBioPortal/cbioportal/pull/4057

File: src/shared/api/generated/CBioPortalAPI-docs.json (hunk 2 above)
Comment (inodb) [reply src/shared/api/generated/CBioPortalAPI-docs.json]: yes length is removed: https://github.com/cBioPortal/cbioportal/pull/6312

**Pull request merged**